cypress.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = defineConfig({
     setupNodeEvents(on, config) {
       on("task", {
         generateUser() {
-          let randomNumber = Math.ceil(Math.random(1000) * 1000);
+          let randomNumber = Math.ceil(Math.random(1000) * 1000000);
           let userName = faker.name.firstName() + `${randomNumber}`
```

Comment:
It is better to use template strings to create complex strings, as it improves readability and maintainability of the code.